developmentseed/scoreboard
# Add/campaign source chips

- Adds chip component
- Adds proptype library - I couldn't find any type checking tool, is it fine to start using proptypes?
- Adjusts card footer style
- Adds color class dynamically based on the enum selected by the tm type. I'm open to better ways of handling this

### mockup:

![image](https://user-images.githubusercontent.com/20410256/90292284-9ca40900-de4f-11ea-8b33-302f72f98d6b.png)

### To Test/Review
navigate to campaigns, filter by tasking manager to see different chip colors
Is there some other tests I should make or run? I see that we have a jest setup but I don't see any tests
